corehq/apps/reports/standard/cases/data_sources.py

```diff
@@ -176,7 +178,17 @@ def _get_username(self, user_id):
         return username
 
     def parse_date(self, date_string):
-        return string_to_utc_datetime(date_string)
+        try:
+            return string_to_utc_datetime(date_string)
+        except:
+            try:
```

Comment:
You could log it couchlog with a tag string in the message and then you can search by the tag string to get the logs